gpm_api/tests/io/test_checks.py

```diff
@@ -0,0 +1,497 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 17 15:41:14 2023
+
+@author: ghiggi
+"""
+
+import pytest
+import datetime
+import numpy as np
+import os
+import platform
+import ntpath as ntp
+import posixpath as ptp
+import pytz
+from typing import List
+from gpm_api.io import checks
+from gpm_api.io.products import available_products, available_scan_modes
+
+
+def test_is_not_empty() -> None:
```

Comment:
I really wonder if we don't need to remove the usage of `is_empty` or `is_not_empty` across the codebase.

Reply:
I took a look and it seems that `is_empty()` is used on lists of filepaths which I believe are of type: List. These can be boolean tested as Python will return False on an empty list.